requirements.txt

```diff
@@ -53,7 +53,7 @@ importlib-resources~=6.1.0
 incremental~=22.10.0
 iniconfig~=2.0.0
 ipykernel~=6.29.0
-ipython~=8.18.0
```

Comment:
I think you need to put the `; python_version` logic here rather than in pyproject.toml